extensions/amp-analytics/0.1/vendors.js

```diff
@@ -698,6 +698,10 @@ export const ANALYTICS_CONFIG = /** @type {!JsonObject} */ ({
           'clt=${contentLoadTime}&' +
           'dit=${domInteractiveTime}' +
           '${baseSuffix}',
+      'user-error': '${host}/collect?${basePrefix}&' +
```

Comment:
maybe call this `exception` or just `error` instead of `user-error`? At the minimum use camelCase so `userError`